easybuild/tools/systemtools.py

```diff
@@ -106,6 +107,28 @@ def get_core_count():
     _log.nosupport("get_core_count() is replaced by get_avail_core_count()", '2.0')
 
 
+def get_total_memory():
+    """
+    Try to ascertain this node's total memory
+    
+    @return: total memory as an integer, specifically a number of megabytes
```

Comment:
If we do that, we should do it consistently (also for other system features), and it should be well documented which environment variables EB picks up on...

So, not in this PR.